### PR TITLE
Feature/organization

### PR DESCRIPTION
**Pull request by HyPn0TiiK**

### CODE REVIEW

**Review by emmadeno — APPROVED** [review]
Great feature !

File: app/src/androidTest/java/com/ncnf/organization/OrganizationTabFragmentTests.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        onView(withId(R.id.user_view_pager)).perform(swipeLeft());
+        onView(withId(R.id.add_organization_button)).perform(click());
+
+        // TODO
+    }
+
+    @Test
+    public void addInvalidOrganization() {
+        // TODO
+    }
```
Comment (eloigrndl) [inline app/src/androidTest/java/com/ncnf/organization/OrganizationTabFragmentTests.java:117]: Test missing : is it normal ?

File: app/src/main/java/com/ncnf/database/DatabaseService.java [hunk 2]
```diff
@@ -140,7 +140,7 @@ private void initRegistry(){
     }
 
     @Override
-    public <T> CompletableFuture<List<T>> withFieldLike(String collectionPath, String field, String value, Class<T> collectionType) {
+    public <T> CompletableFuture<List<T>> withFieldContaining(String collectionPath, String field, String value, Class<T> collectionType) {
```
Comment (eloigrndl) [inline app/src/main/java/com/ncnf/database/DatabaseService.java:143]: Useful renaming !

File: app/src/main/java/com/ncnf/organization/OrganizationTabFragment.java [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+            if (organizations.isEmpty()) {
+                recycler.setVisibility(View.GONE);
+                emptyView.setVisibility(View.VISIBLE);
+            } else {
+                recycler.setVisibility(View.VISIBLE);
+                emptyView.setVisibility(View.GONE);
+            }
+        }
+    }
+*/
```
Comment (eloigrndl) [inline app/src/main/java/com/ncnf/organization/OrganizationTabFragment.java:281]: Should we keep this block if it is not used ?

**Review by eloigrndl — APPROVED** [review]
Important part done here : good job ! Might need several changes later as I saw some TODOs left.

**Pull request closed without merging**